### PR TITLE
New template filters pack and unpack

### PR DESCRIPTION
**Pull request by jbouwh**
## Proposed change

Add documentation for new template filters `pack` and `unpack` filters.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/60836
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by jbouwh** [comment]
Parent closed.

**Comment by jbouwh** [comment]
Parent re-opened

**Comment by jbouwh** [comment]
Should be `next` branch sorry.

File: source/_docs/configuration/templating.markdown [hunk 1]
```diff
@@ -574,6 +574,26 @@ The numeric functions and filters will not fail if the input is not a valid numb
 - Filter `value_one|bitwise_or(value_two, little_endian=False)` perform a bitwise or(\|) operation with two values. If a value is a `bytes` object, it will be translated to an integer in Big-endian style (the first byte is contains the most significant bit). Set `little_endian` to `True` to parse `bytes` object in Little-endian style.
 - Filter `ord` will return for a string of length one an integer representing the Unicode code point of the character when the argument is a Unicode object, or the value of the byte when the argument is an 8-bit string.
 
+### Functions and filters to process raw data
+
+These functions are used to process raw value's in a `bytes` format to values in a native Python type or vice-versa.
+The `pack` and `unpack` functions can also be use as a filter. The make use of the Python 3 `struct` library.
```
Comment (frenck) [inline source/_docs/configuration/templating.markdown]:
```suggestion
The `pack` and `unpack` functions can also be used as a filter. They make use of the Python 3 `struct` library.
```

**Review by frenck — APPROVED** [review]
Thanks, @jbouwh 👍

**Pull request merged**